### DISCUSSION TITLE
Retrieving Texture-coordinates from point (clientX, clientY)

### DISCUSSION THREAD

**Discussion by BR14Nx** [google/model-viewer#3006] [Ideas]
So far the documentation describes methods to retrieve materials, 3D-coordinates and normals ''from the input pixel coordinates relative to the screen''. However, I want to <strong>retrieve</strong> the <strong>UV-coordinates</strong> of the 3D-model to test live painting. (In my case the model has only one texture). 
Is there already such method implemented or is there a workaround?

**Comment by elalish** [comment]
You're the  second user to ask for this now, so we should probably just implement it. It shouldn't be hard. In fact, if you'd be up for making a PR, I'll be happy to review  and merge it. You should be able to follow the code for `positionAndNormalFromPoint`, since three.js  already returns the information you desire.
